4C Integration.md

```diff
@@ -0,0 +1,262 @@
+# 4C integration doc
+
+###### This document will give a brief overview of the MX media-buying platform, and explain how to interact with its API.
+
+### Platform Overview
+
+#### Domain Concepts
+
+Within the MX platform we think in terms of `Organizations`, `Users`, `Teams`, `Roles`, `Markets`, `Orders`, `Assets`, `Attributes`, and `Matches`.
+
+* An `Organization` represents a single buyer or seller, and is analogous to a company account. There are many `Users` within an `Organization`, and any action a user takes looks from the outside as if it was done by the organization. Users are assigned to `Teams`, and given `Roles` within those teams which constrain which actions they are allowed to take.
+
+* A `Market` is an environment in which transactions happen. There is an `Open Market` and several `Private Markets`. Initially, all transactions will happen in the latter. `Orders` in private markets can only be seen by market members, which are `Organizations`, and defined by the market owner.
```

Comment:
maybe say 'user made' instead of 'several'. 'several' seems finite and it will help with 'defined by market owner' later in the paragraph